src/replicache.ts

```diff
@@ -257,6 +257,24 @@ export default class Replicache implements ReadTransaction {
     }
   }
 
+  async createIndex(def: {
+    name: string;
+    keyPrefix?: string;
+    jsonPointer: string;
+  }): Promise<void> {
+    const name = `.rep_internal_create_index_${def.name}`;
```

Comment:
These mutation functions gets sent to the server too. I wonder if we should provide an internal way to mark these or if we should reserve names starting with something.

Reply:
Better than that: I'm going to make them go into a new kind of commit that doesn't get sent to the server as part of https://github.com/rocicorp/repc/issues/218. Should happen in the next couple of days.

Reply:
done in https://github.com/rocicorp/repc/pull/260